docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -439,15 +440,104 @@ spec:
   deny: {}
 ```
 
-##### Access Requests for Kubernetes resources
+##### Preventing unintended access
```

Comment:
This should be also moved.
I think the H5 was wrong and it should be inside Access Requests for Kubernetes resources

Reply:
From my understanding of the guide, the "Preventing unintended access" section has to do with defining a role, while the "Request access to Kubernetes resources" section has to do with `tsh` commands. As a result, I think "Preventing unintended access" belongs in the "Kubernetes resources" H4, which is about defining a role.

What do you think?

Reply:
👍 Fixed in 3163aed0e4c6e1bc4b18a8df2ffbe8888fd2e3e0